### PR TITLE
Fix for computer with multi-interfaces

### PR DESCRIPTION
**Pull request by rmath**
Using HostName with JmmDNS + rejecting all queries / responses coming
from an other interface

### CODE REVIEW

File: src/main/java/javax/jmdns/impl/JmmDNSImpl.java [hunk 1]
```diff
@@ -85,11 +87,20 @@
...
+    private String                                              _defaultHostName = null;
+
     /**
      *
      */
     public JmmDNSImpl() {
         super();
+
+        try {
+            _defaultHostName = InetAddress.getLocalHost().getHostName();
```
Comment (spearway) [inline src/main/java/javax/jmdns/impl/JmmDNSImpl.java]: This actually makes a DNS query which may not be advisable in this class.

Comment (rmath) [inline src/main/java/javax/jmdns/impl/JmmDNSImpl.java]: Thanks for your response. I will try another way to set the same name for all JmDNS instances of the same computer.

Comment (kaikreuzer) [inline src/main/java/javax/jmdns/impl/JmmDNSImpl.java]: @rmath Do you still plan to do an update here?

**Comment by kaikreuzer** [comment]
Closing due to inactivity.

**Pull request closed without merging**